### ISSUE TITLE
Very negative timestamp can cause undefined behavior sanitizer to trigger

### ISSUE THREAD

**Issue by nyh** [scylladb/scylla#8895]
This problem was detected in Jenkins run https://jenkins.scylladb.com/job/scylla-master/job/next/3664/consoleText

The cql-pytest test translated from Cassandra `cassandra_tests/validation/entities/timestamp_test.py`::`testNegativeTimestamps` (temporarily in "next" but now dequeued) detected a sanitizer error in the debug build:

[2021-06-21T08:54:34.257Z] cql3/util.cc:129:23: runtime error: signed integer overflow: -9223372036854775808 - 1624262309713000 cannot be represented in type 'long'
[2021-06-21T08:54:34.257Z] SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior cql3/util.cc:129:23 in 

the test line exponsing this undefined behavior was:
```
assert_invalid(cql, table, "INSERT INTO %s (k, v) VALUES (?, ?) USING TIMESTAMP ?", 2, 2, -2**63)
```

**Comment by nyh** [comment]
Note that since #5619 was fixed by commit 2150c0f7a2d56b9ea2a5607a4066c4c1c3aba40, we don't allow any signed 64-bit timestamp - any number more than 3 days into the future is refused (the test for this is exactly what has undefined behavior because it subtracts two signed 64-bit integers).
So arguably, under the same logic - that timestamps should have a concrete meaning (actual time in microseconds) -  negative timestamps should also be disallowed.
But this may be beyond the cope of this issue. Or not...

**Comment by nyh** [comment]
Already fixed by ec5e4c338b38d56d37e427b9abcbacf86a683b84. So closing issue.
